cloud-sql/sqlserver/mssql/server.js

```diff
@@ -0,0 +1,197 @@
+// Copyright 2019 Google LLC
```

Comment:
```suggestion
// Copyright 2020 Google LLC
```

Don't forget to change this everywhere

Reply:
did a global find/replace to change all of them.